src/core_plugins/kibana/public/discover/controllers/discover.js

```diff
@@ -144,7 +144,8 @@ function discoverController($scope, config, courier, $route, $window, Notifier,
   $scope.indexPattern = resolveIndexPatternLoading();
   $scope.searchSource
   .set('index', $scope.indexPattern)
-  .highlightAll(true);
+  .highlightAll(true)
+  .version(true);
```

Comment:
Absolutely, good catch. I also realized I needed to update the `track by` above the first one I changed. Updated.